holoviews/plotting/bokeh/element.py

```diff
@@ -1832,7 +1838,20 @@ def _get_color_data(self, element, ranges, style, name='color', factors=None, co
 
     def _get_cmapper_opts(self, low, high, factors, colors):
         if factors is None:
-            colormapper = LogColorMapper if self.logz else LinearColorMapper
+            if self.logz:
+                colormapper = LogColorMapper
+                if util.is_int(low) and util.is_int(high) and low == 0:
+                    low = 1
+                    if 'min' not in colors:
+                        colors['min'] = 'rgba(0, 0, 0, 0)'
+                elif util.is_number(low) and low == 0:
+                    self.param.warning(
+                        "Log color mapper lower bound is 0, ensure you "
+                        "set a non-zero lower bound on the color dimension "
+                        "or using the `clim` option."
```

Comment:
As a slight tweak, the lower bound isn't *supposed* to be zero but that is our heuristic/guess. So maybe something like:

> 'Unspecified log color mapper lower bound defaulting to 0, ....'

Reply:
There are no assumptions being made and in all likelihood everything will be white since there is no particularly principled way to pick a lower bound.